backend/src/util/index.ts

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+export async function to<T>(p: Promise<T>) {
```

Comment:
Just sayin' that I still disagree with the existence of this one, but if you really like it... :p

Reply:
Refactored, and file removed